code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -421,7 +421,8 @@
 	icon = 'icons/obj/machines/sleeper.dmi'
 	icon_state = "sleeper_s"
 	outfit = /datum/outfit/syndicate_empty
-	assignedrole = "Space Syndicate"	//I know this is really dumb, but Syndicate operative is nuke ops
+	spawner_special_role = ROLE_KEY_EXT_SYNDI_AGENT	//I know this is really dumb, but Syndicate operative is nuke ops
+											// Evildragon: I don't get this comment ↑
```

Comment:
`#define ROLE_KEY_NUCLEAR_OPERATIVE "Syndicate Operative"`
the name is already taken, so they made it "Space Syndicate" instead of what they wanted ("Syndicate Operative")

just remove the comment now

Reply:
I think I got what they wanted.